### ISSUE TITLE
Ubuntu 20.04 LTS asks to restart and switch off the intel RST

### ISSUE THREAD

**Issue by Filipedls** [kfechter/LegionY530Ubuntu#99]
**Describe the bug**
During the installation of Ubuntu I had to restart due to the Intel RST hdd driver.

**To Reproduce**
Followed exactly the same steps in the tutorial, but for a Lenovo Y720

**Expected behavior**
Got the Ubuntu installation screen asking to restart and switch off the intel RST.

**Screenshots**

**System Info (please complete the following information):**
 - Ubuntu 20.04LTS 
 - Lenovo Y720

**Solution:
in UEFI/BIOS, switch SATA controller from Intel RST Premium to AHCI

**Comment by kfechter** [comment]
RST isn't very well supported, and is generally only enabled on machines with a hard drive and a cache ssd, or in the case of newer models, Optane. 

I can add some additional information to the troubleshooting steps. I cannot grab screenshots though because I don't think my Y530 is configured in a way that allows iRST.

**Comment by kfechter** [comment]
I have updated the guide to include this information in the considerations section and the troubleshooting section.

**Comment by Filipedls** [comment]
Hey! Thanks, sorry for the late response, but let me know if you want me to take some pictures of the Y720 bios, even though it's not the same PC, this guide was quite useful for me still. I also had to disable some safeboot options in windows, windows doesnt like to boot with a different sata controller. I had to do those steps: https://support.thinkcritical.com/kb/articles/switch-windows-10-from-raid-ide-to-ahci. This is easily available if you install windows with AHCI mode from the start.
